websites/M/Mastodon/metadata.json

```diff
@@ -0,0 +1,33 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "533014724569333770",
+		"name": "Hikki"
+	},
+	"service": "Mastodon",
+	"description": {
+		"en": "From breaking news to entertainment, from technology to politics, mastodon is a social that empowers users without tracking them."
+	},
+	"url": [
+		"mastodon.social",
+		"mstdn.social",
+		"techhub.social",
+		"mastodon.uno",
+		"securitymastod.one",
+		"linuxrocks.online",
+		"universeodon.com",
+		"indieweb.social",
+		"infosec.exchange",
+		"aus.social",
+		"social.network.europa.eu"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/B0HPMOQ.png",
+	"thumbnail": "https://i.imgur.com/e7gLIsD.jpeg",
```

Comment:
This thumbnail is a bit low-res. Try to find a higher-resolution version or upscale it if possible.